docs/proposals/opaque-integer-resources.md

```diff
@@ -0,0 +1,127 @@
+# Accounting and Consuming Opaque Integer Resources
+
+## Terms
+
+### Opaque node-level resource
+A resource associated with a node that has a consumable capacity, which is handled uniformly by the scheduler and Kubelet. (If there are specializations that hang off of the resource name, then it is not opaque!)
+
+## Requirements
+- Associate available opaque counted resources with existing nodes.
```

Comment:
Maybe a numbered list would be better here so people can refer to the requirements that way.